website/src/components/Changelog/GUI.tsx

```diff
@@ -22,19 +28,24 @@ export default function GUI({ title }: { title: string }) {
               a bug where DNS could stop working when you sign out.
             </Link>
           </li>
+          )}
         </ul>
       </Entry>*/}
       <Entry version="1.1.5" date={new Date("2024-07-08")}>
         <ul className="list-disc space-y-2 pl-4 mb-4">
-          <li className="pl-2">
-            The Linux GUI Client is now{" "}
+          {title === "Linux GUI" && (
+          <p>The Linux GUI Client is now{" "}
             <Link
               href="https://github.com/firezone/firezone/pull/5793"
               className="text-accent-500 underline hover:no-underline"
             >
               built for both x86-64 and ARM64
             </Link>
-          </li>
+          </p>
+          )}
+          {title === "Windows" && (
```

Comment:
The Windows changelog currently says "Linux is now built for both..." so this hides that